### ISSUE TITLE
Importing full v13 pack into v15 misses a lot of templates

### ISSUE THREAD

**Issue by PhyxionNL** [Jumoo/uSync.Complete.Issues#248]
**Describe the bug**

**Version (please complete the following information):**
 - Umbraco Version: 15.0
 - uSync Version: 15.0
 - uSync.Complete Version: 15.0

**To Reproduce**
1. Export a full sync pack in uSync in v13 installation.
2. Create v15 project.
3. Import v13 into v15.
4. I only get 5 templates in Umbraco. All the other content seems to be imported OK. Weirdly it does report 70 templates in the progress thingy, but only 5 are added.

**Expected behavior**
Get all templates. It works OK to import v13 pack into a blank v13 and then upgrade Umbraco.

**Comment by KevinJump** [comment]
Hi, 

Can you send a screen shot of the structure ? is there any noticible logic, (it might be failing on children for example?)

**Comment by KevinJump** [comment]
Also does a second import bring them in (just wondering).

**Comment by PhyxionNL** [comment]
> Hi,
> 
> Can you send a screen shot of the structure ? is there any noticible logic, (it might be failing on children for example?)

I didn't find any logic, I've tried it a couple of times and it are always the same items that are imported.
When it's processing "Content", I do see a lot of these in console:

[13:10:34 INF] The reference to umb://media/695a51117c694d1e8629451d9acf1ab7 can not be saved as relation, because doesn't have a node ID.
[13:15:16 INF] The reference to umb://document/9aa809248d6d41d69124bf3be81ee9cb can not be saved as relation, because doesn't have a node ID.
[13:15:18 WRN] no json images found for []

Screenshot:
![image](https://github.com/user-attachments/assets/d71f9008-dee6-4dee-97a0-73d075b851aa)

It says all these templates are imported, but they're not listed in Umbraco under Templating > Templates, and not bound to the Document Types, so Umbraco complains about missing templates.

The templates in the exported zip file look the same compared to the ones that are imported.

> Also does a second import bring them in (just wondering).

Sadly not, it has the same issue. I also tried Import (Force).

**Comment by KevinJump** [comment]
yeah - you can disregard the relation errors (that's an umbraco issue, it happens when you install the starter kit on a blank site) 😞 

I will take a look see if this is anything i can reproduce.

**Comment by KevinJump** [comment]
Its been a few versions, and it was hard for me to get to the bottom of this one, I think some of the changes in the v15.1.2 release might have helped (we had Template path and template master issues).
